packages/expo-cli/README.md

```diff
@@ -30,7 +30,19 @@ To view this on your phone, do the following:
 To publish something you've made, just follow these steps:
 
 - Create an Expo account or login to an existing one by running `expo login`.
-- Run an Expo CLI server using `expo start`.
+- Run an Expo CLI server usingTo run your project, navigate to the directory and run one of the following yarn commands.
```

Comment:
This kinda breaks the list formatting. Also `yarn start` could mean anything, it doesn't always map to `expo start`